src/RA_AchievementOverlay.cpp

```diff
@@ -1499,6 +1493,13 @@ void AchievementOverlay::InstallNewsArticlesFromFile()
     }
 }
 
+void AchievementOverlay::UpdateImages() noexcept
+{
+    using ra::services::ImageReference;
+    m_hOverlayBackground = ImageReference{ ra::services::ImageType::Local, "Overlay\\overlayBG.png" };
+    m_hUserImage         = ImageReference{ ra::services::ImageType::UserPic, RAUsers::LocalUser().Username() };
```

Comment:
These'll use `ra::services::ImageReference`'s `move assignment`, it moves it from a temporary (`rvalue reference`).